wallet-new/src/Cardano/Wallet/API/V1/Generic.hs

```diff
@@ -1,149 +1,37 @@
 {-# LANGUAGE PolyKinds #-}
 
 module Cardano.Wallet.API.V1.Generic
-       ( gtoJsend
-       , gparseJsend
-       , gconsNames
+       ( gconsNames
        , gconsName
        ) where
 
 import           Universum hiding (All, Generic)
 
-import           Data.Aeson
-import           Data.Aeson.Types (Parser)
-import qualified Data.HashMap.Strict as HM
-import qualified Data.Vector as V
-import           Generics.SOP
-import           Generics.SOP.JSON (JsonInfo (..), JsonOptions (..), Tag (..),
-                     defaultJsonOptions, jsonInfo)
-
-import           Cardano.Wallet.API.Response.JSend (ResponseStatus (..))
-import           Cardano.Wallet.Util (mkJsonKey)
 import           Data.List ((!!))
-import           Pos.Util.Util (aesonError)
-
---
```

Comment:
Now that `WalletError` JSON instances are defined by hand, we don't need all the sop-generic stuff. I'd be more than happy to re-introduce generics here if we can find a way to do so without relying on partial field accessors.